client/my-sites/media-library/external-media-header.jsx

```diff
@@ -84,13 +88,42 @@ class MediaLibraryExternalHeader extends Component {
 	}
 
 	onCopy = () => {
-		const { postId, site, selectedItems, source, onSourceChange } = this.props;
+		const {
+			postId,
+			site,
+			selectedItems,
+			source,
+			onSourceChange,
+			photosPickerSession,
+			photosPickerApiEnabled,
+		} = this.props;
 
 		onSourceChange( '', () => {
 			this.props.addExternalMedia( selectedItems, site, postId, source );
+			photosPickerApiEnabled && this.props.deletePhotosPickerSession( photosPickerSession?.id );
```

Comment:
I'm thinking out loudly here. After some more thoughts, I'm wondering if we shouldn't delete the session when copy happens. Reason being that users might select all the needed media at one time, but just select and copy a few at a time, and might have several pages to go through. It'll be tough for them to reselect it every time. Maybe we can keep the session, until it's expired or user selects "change selection". What do you think?

Reply:
Hmmm, not sure. 🤔 

If the user spends some time making a selection, he copies all of them to the media library in the second step and then consumes those files from the media library in the different flows, posts/pages, etc.

Honestly, I don't have a strong preference here.

Reply:
> If the user spends some time making a selection, he copies all of them to the media library in the second step and then consumes those files from the media library in the different flows, posts/pages, etc.

Could you help me understand this more? They can consume the files from the media library in the different flows later, but I don't see how this will be different compare to the current flow.

The reason I was wondering about this is because if user have more than one page of selection, they probably can't select all at once, and have to go back to the Google photos page after they click the "Copy to media library" button again. 🤔  Does that make sense?

Reply:
> Does that make sense?

I didn't think from that perspective. Then, it makes sense.
I'll remove this code line; there will be a way to renew the session by clicking on the "Change selection" button.

Reply:
Does it make sense to call `deletePhotosPickerSession` to delete the current session when user clicks  "Change selection" button?